src/models/applicationState.ts

```diff
@@ -158,7 +158,8 @@ export interface IRegion {
     id: string;
     type: RegionType;
     tags: ITagMetadata[];
-    points: IPoint[];
+    points?: IPoint[];
```

Comment:
@PIC123 - Let's discuss how we handle the different region types.

Reply:
@JacopoMangiavacchi

Reply:
@PIC123 @JacopoMangiavacchi Will refactor in seperate PR based on our share/points/region discussion this morning.